components/SEO.tsx

```diff
@@ -15,11 +15,11 @@ export default function SEO({
     let openGraph = {
         title: fullTitle,
         description: description,
-        url: "https://your-domain.com" + router.asPath,
+        url: "https://hosatinder.vercel.app" + router.asPath,
```

Comment:
what does this do?

Reply:
I think it's just so the search engine optimization stuff knows that the URL of the website is? it was part of samson's template